torch/quantization/fx/_equalize.py

```diff
@@ -251,8 +252,19 @@ def get_weight_eq_obs(node: Node, model: GraphModule, modules: Dict[str, nn.Modu
         return node, weight_eq_obs
 
     elif node.op == 'call_function':
-        # TODO
-        return None, None
+        # If the next node is a functional layer, then the weight observer node
+        # has already been created, so we want to traverse the weight observer
+        # nodes to find the WeightEqualizationObserver node
```

Comment:
it would be great if the behavior was consistent for modules and functionals

Reply:
I agree, I tried to make the code as consistent as possible, but I couldn't find a way to differentiate between the modules and the functional layers. The difference is that for modules we need to create the weight equalization observer since it has not been created yet. This can be done by checking in the `equalization_config_map` for if the layer needs to be equalized, and then initializing an observer.

However, for functional layers, the weight equalization observer node has already been created, so we can just look in the graph for the particular weight node and return that. I think my code here is a little over-complicated and there is an easier addition.

Reply:
I updated the code due to the changes to `prepare`. Let me know what you think!